### PR TITLE
[FLINK-11558] Translate "Ecosystem" page into Chinese

### PR DESCRIPTION
**Pull request by kaibozhou**
Translate "Ecosystem" page into Chinese.

The markdown file is located in: flink-web/ecosystem.zh.md
The url link is: https://flink.apache.org/zh/ecosystem.html

### CODE REVIEW

**Comment by XuQianJin-Stars** [comment]
hi @kaibozhou https://zeppelin.incubator.apache.org/ should be changed to https://zeppelin.apache.org/
best
qianjin

File: ecosystem.zh.md [hunk 1]
```diff
@@ -25,78 +24,70 @@ many other data processing projects and frameworks.
...
-See Suneel Marthi's [Flink Forward talk](http://www.slideshare.net/FlinkForward/suneel-marthi-bigpetstore-flink-a-comprehensive-blueprint-for-apache-flink?ref=http://flink-forward.org/?session=tbd-3) as preview.
+[BigPetStore](https://github.com/apache/bigtop/tree/master/bigtop-bigpetstore) 是一个包含数据生成器的基准测试套件，很快就可以用于 Flink 。 请参阅 Suneel Marthi 在 [Flink Forward 会议上的演讲](http://www.slideshare.net/FlinkForward/suneel-marthi-bigpetstore-flink-a-comprehensive-blueprint-for-apache-flink?ref=http://flink-forward.org/?session=tbd-3)。
 
 **FastR**
 
-[FastR](https://github.com/oracle/fastr) is an implemenation of the R language in Java. [FastR Flink](https://bitbucket.org/allr/fastr-flink/src/3535a9b7c7f208508d6afbcdaf1de7d04fa2bf79/README_FASTR_FLINK.md?at=default&fileviewer=file-view-default) executes R workloads on top of Flink.
+[FastR](https://github.com/oracle/fastr) 是 Java 中 R 语言的实现。 [FastR Flink](https://bitbucket.org/allr/fastr-flink/src/3535a9b7c7f208508d6afbcdaf1de7d04fa2bf79/README_FASTR_FLINK.md?at=default&fileviewer=file-view-default) 在 Flink 之上执行 R 任务。
```
Comment (wuchong) [inline ecosystem.zh.md]: 在 -> 可以在  ?

File: ecosystem.zh.md [hunk 2]
```diff
@@ -25,78 +24,70 @@ many other data processing projects and frameworks.
...
 
-[Alluxio](http://www.alluxio.org/) is an open-source memory-speed virtual distributed storage that enables applications to efficiently share data and access data across different storage systems in a [unified namespace](http://www.alluxio.org/docs/master/en/Unified-and-Transparent-Namespace.html). Here is an example of [using Flink to access data through Alluxio](http://www.alluxio.org/docs/master/en/Running-Flink-on-Alluxio.html).
+[Alluxio](http://www.alluxio.org/) 是一个开源的，能匹配内存速度的虚拟分布式存储，使应用程序能够在[统一的命名空间](http://www.alluxio.org/docs/master/en/Unified-and-Transparent-Namespace.html)中有效地共享数据并跨不同存储系统访问数据。以下是[使用 Flink 通过 Alluxio 访问数据的示例](http://www.alluxio.org/docs/master/en/Running-Flink-on-Alluxio.html)。
 
 **Python Examples on Flink**
 
-A [collection of examples](https://github.com/wdm0006/flink-python-examples) using Apache Flink's Python API.
+使用 Apache Flink 的 Python API 的[一组示例](https://github.com/wdm0006/flink-python-examples)。
 
 **WordCount Example in Clojure**
```
Comment (wuchong) [inline ecosystem.zh.md]: Should we translate this ?

`**使用 Clojure 编写的 WordCount 示例**` ?

File: ecosystem.zh.md [hunk 3]
```diff
@@ -25,78 +24,70 @@ many other data processing projects and frameworks.
...
 
-A small [WordCount example](https://github.com/mjsax/flink-external/tree/master/flink-clojure) on how to write a Flink program in Clojure.
+关于如何在 Clojure 中编写 Flink 程序的 [WordCount示例](https://github.com/mjsax/flink-external/tree/master/flink-clojure)。
 
 **Anomaly Detection and Prediction in Flink**
```
Comment (wuchong) [inline ecosystem.zh.md]: `**异常检测和预测**` ?

File: ecosystem.zh.md [hunk 4]
```diff
@@ -25,78 +24,70 @@ many other data processing projects and frameworks.
...
 
-[flink-htm](https://github.com/nupic-community/flink-htm) is a library for anomaly detection and prediction in Apache Flink. The algorithms are based on Hierarchical Temporal Memory (HTM) as implemented by the Numenta Platform for Intelligent Computing (NuPIC).
+[flink-htm](https://github.com/nupic-community/flink-htm) 是 Apache Flink 中用于异常检测和预测的库。该算法基于由 Numenta 智能计算平台（NuPIC）实现的 Hierarchical Temporal Memory（HTM）。
 
 **Apache Ignite**
 
-[Apache Ignite](https://ignite.apache.org) is a high-performance, integrated and distributed in-memory platform for computing and transacting on large-scale data sets in real-time. See [Flink sink streaming connector](https://github.com/apache/ignite/tree/master/modules/flink) to inject data into Ignite cache.
+[Apache Ignite](https://ignite.apache.org) 是一个高性能，集成和分布式的内存计算和事务平台，用于实时处理大规模数据集。请参阅 [Flink sink streaming connector](https://github.com/apache/ignite/tree/master/modules/flink) 以将数据写入 Ignite 缓存。
 
 **Tink temporal graph library**
```
Comment (wuchong) [inline ecosystem.zh.md]: `**Tink 时态图库**`?

File: ecosystem.zh.md [hunk 5]
```diff
@@ -25,78 +24,70 @@ many other data processing projects and frameworks.
...
-This is a list of third party packages (i.e., libraries, system extensions, or examples) built on Flink.
-The Flink community collects links to these packages but does not maintain them.
-Thus, they do not belong to the Apache Flink project, and the community cannot give any support for them.
-**Is your project missing?**
-Please let us know on the [user/dev mailing list](#mailing-lists).
+这是基于 Flink 构建的第三方软件包（包括库，系统扩展或示例）的列表。
+ Flink 社区收集这些包的链接，但不维护它们。
+因此，它们不属于 Apache Flink 项目，社区无法为它们提供任何支持。
+**是否遗漏了您的项目？**
+请通过 [user/dev mailing list](#mailing-lists) 告诉我们。
```
[the remark this answers is not in the document]
Comment (wuchong) [reply ecosystem.zh.md]: The mailing list link is broken. The correct link should be `{{ site.baseurl }}/zh/community.html#mailing-lists`. The corresponding English link should be `{{ site.baseurl }}/community.html#mailing-lists`.

[the remark this answers is not in the document]
Comment (kaibozhou) [reply ecosystem.zh.md]: Good catch！The link is broken.

File: ecosystem.zh.md (hunk 2 above)
Comment (kaibozhou) [reply ecosystem.zh.md]: +1

File: ecosystem.zh.md (hunk 3 above)
Comment (kaibozhou) [reply ecosystem.zh.md]: +1

File: ecosystem.zh.md (hunk 4 above)
Comment (kaibozhou) [reply ecosystem.zh.md]: This title can be translated to be consistent.

**Comment by kaibozhou** [comment]
@XuQianJin-Stars @wuchong Thanks for the review, all comments has addressed.

**Comment by wuchong** [comment]
Thank you @kaibozhou . LGTM. 

I will merge this soon.

**Pull request closed without merging**